spin.toml

```diff
@@ -7,13 +7,13 @@ name = "crud-list"
 version = "0.1.0"
 
 [[trigger.http]]
-route = "/messages"
+route = "/v1/messages/..."
```

Comment:
Make sure you change the project folder structure to include the v1 folder as well.